common/annotations/en.xml

```diff
@@ -18,15 +18,15 @@ annotations.
 	<annotations>
 		<annotation cp="{">brace | bracket | curly brace | curly bracket | gullwing | open curly bracket</annotation>
 		<annotation cp="{" type="tts">open curly bracket</annotation>
-		<annotation cp="🏻">light skin tone | skin tone | type 1–2</annotation>
+		<annotation cp="🏻">1–2 | light | skin | tone | type</annotation>
```

Comment:
1-2 by itself seems unexpected?

Reply:
Updated to '| type 1-2 | light | skin tone |' in spreadsheet (did the same for the other skin tone lines)